catalogue/webapp/pages/work.js

```diff
@@ -7,34 +7,37 @@ import {font, spacing, grid, classNames} from '@weco/common/utils/classnames';
 import {convertImageUri, iiifImageTemplate} from '@weco/common/utils/convert-image-uri';
 import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
 import InfoBanner from '@weco/common/views/components/InfoBanner/InfoBanner';
+import {workLd} from '@weco/common/utils/json-ld';
+import WorkMedia from '@weco/common/views/components/WorkMedia/WorkMedia';
```

Comment:
What's the reason for reordering this?

Reply:
There isn't a good reason – I removed the imports when they weren't being used when I initially moved everything into the single column component. Then I had to re-add them when we agreed to do this behind a toggle.